src/runner/bootstrapper.ts

```diff
@@ -392,4 +394,12 @@ export default class Bootstrapper {
 
         return { ...await this._bootstrapSequence(this.browsers, id), commonClientScripts, id };
     }
+
+    public restoreTestRunEvents (): void {
+        const connections = this.browserConnectionGateway.getConnections();
+
+        Object.values(connections).forEach(connection => {
+            connection.initMessageBusEvents();
```

Comment:
the `initMessageBusEvents` method does `emit('message-bus-initialized')` event. Is not this a problem here?
Should not we just reassign the `test-run-start` handler?

Reply:
the `initMessageBusEvents` is still to complecated. Can we simplify it?